src/client/components/localisation/network.ts

```diff
@@ -0,0 +1,45 @@
+import { action } from 'mobx'
+import { message } from '../../../shared/proto/messages'
```

Comment:
The problem an alias is that IntelliJ cannot automatically add the import for you, I'd rather never type an import if I don't have to 😛